common/decimal_helpers.go

```diff
@@ -0,0 +1,42 @@
+package common
+
+import (
+	"github.com/shopspring/decimal"
+)
+
+const sqrtMaxIterations = 10000
+
+// Float returns the decimal value as a float64
+// float is a helper function for float64() and doesnot return bool exact
+func Float(d decimal.Decimal) float64 {
+	f, _ := d.Float64()
+	return f
+}
+
+// NotEqual returns true when d is not equal to d2
+func NotEqual(d, d2 decimal.Decimal) bool {
+	return !d.Equal(d2)
+}
+
+//NewFromInt returns a decimal with the value of int v
+func NewFromInt(v int) decimal.Decimal {
```

Comment:
Change naming convention either decimal.NewFromInt() or common.NewDecimalFromtInt(101) to be a bit more specific across this package.